### PR TITLE
Fix rust dependencies on Windows

### PR DESCRIPTION
**Pull request by MonsieurNicolas**

### CODE REVIEW

**Review by dmkozh — COMMENTED** [review]
LGTM, modulo a minor nitpick

File: src/util/Decoder.h [hunk 1]
```diff
@@ -48,26 +44,6 @@ encode_b64(T const& v)
     return res;
 }
 
-#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
```
[the remark this answers is not in the document]
Comment (MonsieurNicolas) [reply src/util/Decoder.h:51]: correct this was added early on with no use cases. We can add something like that (in a different header) if we need it in the future

[the remark this answers is not in the document]
Comment (graydon) [reply src/util/Decoder.h:51]: If you want to remove the base64 stuff you might as well cut it out of the rust side too, but fine by me.

File: src/simulation/LoadGenerator.cpp [hunk 2]
```diff
@@ -770,8 +770,8 @@ LoadGenerator::createTransactionFramePtr(
         // some_int`. This also would exercise more code paths/logic during the
         // transaction comparisons.
         auto fractionalFeeDistr =
-            uniform_int_distribution<uint32_t>(0, ops.size() - 1);
-        fee = ops.size() * feeRateDistr(gRandomEngine) +
+            uniform_int_distribution<uint32_t>(0, uint32_t(ops.size()) - 1);
```
[the remark this answers is not in the document]
Comment (MonsieurNicolas) [reply src/simulation/LoadGenerator.cpp]: agree. fixed

**Comment by MonsieurNicolas** [comment]
r+ 0aaef62a3cf25a622066aa24885a274801a6d0d5

**Pull request merged**